air-quality-ui/system_tests/ui/summary_page/summary_page_table_data.spec.ts

```diff
@@ -921,3 +921,114 @@ test.describe('Multiple pollutants share the max diff, and have the SAME overall
     await summaryPage.assertGridAttributes('values', expectedTableContents)
   })
 })
+
+test('Verify changing date in date picker triggers change of data displayed', async ({
```

Comment:
Do we gain much from this test? 

As we're mocking the data all we're actually checking here is that updating the datetime causes a fresh API call to be made, which I think is basically covered elsewhere?

Reply:
My concern here was covering off the question 'can the UI update with a response from the datepicker API call'.